diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -178,7 +178,7 @@ class BlockStructureBraces(private val configRules: List<RulesConfig>) : Rule("b
                 if (braceSpace == null || braceSpace.elementType != WHITE_SPACE) {
                     node.addChild(PsiWhiteSpaceImpl(" "), nodeBefore)
                 } else {
-                    (braceSpace as LeafPsiElement).replaceWithText(" ")
+                    braceSpace.treeParent.replaceWhiteSpaceText(braceSpace, " ")
```

Comment:
here can be problems if  treeParent is not a WhiteSpace or does not exist, need to be careful with that

Reply:
But above, there is a check for this case `checkBraceNode`